src/app/onion/old-learning-object-builder/learning-object.resolver.ts

```diff
@@ -10,14 +10,15 @@ export class LearningObjectResolve implements Resolve<LearningObject> {
     private loService: LearningObjectService
   ) {}
 
+
   resolve(route: ActivatedRouteSnapshot) {
     if (route.params['learningObjectName']) {
       const name = route.params['learningObjectName'];
       return this.loService.getLearningObject(name).then(learningObject => {
         return learningObject;
-        }).catch(err => {
-          return;
-        });
+      }).catch(err => {
+        return undefined;
```

Comment:
What is the purpose of this? A comment might be helpful

Reply:
This is an unused file and will be removed